content/participants/ttjc@mann01/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Mann Jain
+type: "participant"
+name: "Mann Jain"
+portfolio: ""
+github-link: "https://github.com/mann01"
+linkedin-link: "https://linkedin.com/in/mann-jain-602bb9163"
+looking-for: "internship"
+description: "Web-Dev | ML | OpenCV | DS-Algo"
+---
+
+Hi! I am a Mann Jain ,3rd year undergraduate let me walk you through my intro, projects and blogs.
```

Comment:
a Mann Jain?? Correct the typos

Reply:
Ya I have modified the things in #276 please review!